packages/FormElement/stories/FormElement.examples.stories.js

```diff
@@ -10,10 +9,11 @@ import DatePicker from "./examples/DatePicker";
 import ListBox from "./examples/ListBox";
 import ButtonGroup from "./examples/ButtonGroup";
 import NativeInput from "./examples/NativeInput";
-import Nested from "./examples/Nested";
+import Nested from "./examples/NestedExample";
```

Comment:
I've been trying to move away from the unnecessary `Example` suffix on these story components.